app/src/main/java/com/hedvig/app/feature/embark/passages/externalinsurer/ExternalInsurerViewModel.kt

```diff
@@ -2,27 +2,39 @@ package com.hedvig.app.feature.embark.passages.externalinsurer
 
 import androidx.lifecycle.ViewModel
 import androidx.lifecycle.viewModelScope
+import kotlinx.coroutines.channels.Channel
 import kotlinx.coroutines.flow.MutableStateFlow
 import kotlinx.coroutines.flow.StateFlow
+import kotlinx.coroutines.flow.asStateFlow
+import kotlinx.coroutines.flow.receiveAsFlow
 import kotlinx.coroutines.flow.update
 import kotlinx.coroutines.launch
 
 class ExternalInsurerViewModel(
     private val getInsuranceProvidersUseCase: GetInsuranceProvidersUseCase
 ) : ViewModel() {
 
+    private var insuranceProviders: List<InsuranceProvider> = emptyList()
```

Comment:
Is there a way to model this where we're not using `var` here? I'm always a bit wary of seeing `var` 😧

Reply:
Perhaps the `insuranceProviders` could always be a part of the `ViewState`, and then the interaction of launching the bottom sheet would never require a round trip to the `ViewModel` 🤔